lib/ruby_lsp/ruby_lsp_rails/rails_test_style.rb

```diff
@@ -0,0 +1,104 @@
+# typed: strict
+# frozen_string_literal: true
+
+module RubyLsp
+  module Rails
+    class RailsTestStyle < Listeners::TestDiscovery
+      extend T::Sig
```

Comment:
I don't think we need to `extend T::Sig` since we're only using comment signatures.